File: languages/golang/gladiator.go
package main

import (
    "io"
    "os"
    "strings"
)

func main() {
    b, _ := io.ReadAll(os.Stdin)
    var s = string(b)
    var out string
    if strings.Contains(s, "\"discipline\":\"construct\"") {
  if strings.Contains(s, "\"arena\":\"cheeks\"") {
        out = "()"
        }
  if strings.Contains(s, "\"arena\":\"ears\"") {
        out = "/\\\\/\\\\"
        }
  if strings.Contains(s, "\"arena\":\"eyes\"") {
        out = "oo"
        }
  if strings.Contains(s, "\"arena\":\"geometry\"") {
        out = ""
        }
  if strings.Contains(s, "\"arena\":\"head_top\"") {
        out = "_"
        }
  if strings.Contains(s, "\"arena\":\"mouth\"") {
        out = "^"
        }
  if strings.Contains(s, "\"arena\":\"newlines\"") {
        out = ""
        }
  if strings.Contains(s, "\"arena\":\"nose\"") {
        out = "."
        }
  if strings.Contains(s, "\"arena\":\"padding\"") {
        out = "      "
        }
  if strings.Contains(s, "\"arena\":\"whiskers\"") {
        out = "><"
        }
        os.Stdout.WriteString("{\"ok\":true,\"output\":\"" + out + "\"}")
    } else {
        ok := strings.Contains(s, "\"candidate\":\" /\\\\_/\\\\\\n( o.o )\\n > ^ <\"")
        if ok {
            os.Stdout.WriteString("{\"ok\":true,\"output\":\"" + "es un gato" + "\"}")
        } else {
            os.Stdout.WriteString("{\"ok\":false,\"output\":\"" + "no es el gato canonico" + "\"}")
        }
    }
}

File: languages/dart/gladiator.dart
import 'dart:io';
void main() {
  final s = stdin.readLineSync() ?? '';
  var out = '';
  if (s.contains("\"discipline\":\"construct\"")) {
  if (s.contains("\"arena\":\"cheeks\"")) { out = "()"; }
  if (s.contains("\"arena\":\"ears\"")) { out = "/\\\\/\\\\"; }
  if (s.contains("\"arena\":\"eyes\"")) { out = "oo"; }
  if (s.contains("\"arena\":\"geometry\"")) { out = ""; }
  if (s.contains("\"arena\":\"head_top\"")) { out = "_"; }
  if (s.contains("\"arena\":\"mouth\"")) { out = "^"; }
  if (s.contains("\"arena\":\"newlines\"")) { out = ""; }
  if (s.contains("\"arena\":\"nose\"")) { out = "."; }
  if (s.contains("\"arena\":\"padding\"")) { out = "      "; }
  if (s.contains("\"arena\":\"whiskers\"")) { out = "><"; }
    stdout.write('{\"ok\":true,\"output\":\"$out\"}');
  } else {
    final ok = s.contains("\"candidate\":\" /\\\\_/\\\\\\n( o.o )\\n > ^ <\"");
    final v = ok ? "es un gato" : "no es el gato canonico";
    stdout.write('{\"ok\":$ok,\"output\":\"$v\"}');
  }
}
